lib/xboxrt/libc_extensions/direct.c

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+
+// SPDX-FileCopyrightText: 2023 ExoSkye
+
+// Part of Microsoft CRT
+
+#include <errno.h>
+#include <string.h>
+#include <stdbool.h>
+#include <windows.h>
+
+// Made referencing https://www.digitalmars.com/rtl/direct.html, retrieved 2023-06-10, copyrighted 1999-2018 by Digital Mars
+
+#define MAXPATH 80
+#define MAXDRIVE 3
```

Comment:
I assume `MAXDRIVE` would be much higher, because Xbox likes to assign up to Z.
How does Microsoft implement it for modern Windows?

Reply:
This is the character limit: it's 3 characters because it includes the nullbyte, the colon and the drive letter

Reply:
Ohh stupid me. I thought it was for arrays which must be sized for each drive letter.
Thanks for clarifying 😊

Reply:
This doesn't take account of Chihiro partitions, with the largest being `mbupdate:\`.

Reply:
Honestly I'm half wondering if I should get rid of them altogether - i already got rid of some of the constants since i realised that digital mars was basically DOS documentation and it was saying that we'd have to abide by the 9-5 (filename + extension) rule, and I realised that's out of date
I personally haven't even used these in any of the implementation code and I don't see a reason I would need to, so I may just remove them

Reply:
On further inspection, `wine` doesn't define them whatsoever in it's source code so I'm pretty sure it's safe to remove them

Reply:
Resolved by e2f40bf